fv3config/_asset_list.py

```diff
@@ -231,6 +165,8 @@ def write_asset(asset, target_directory):
         logger.debug(f"Writing asset bytes to {target_path}.")
         with open(target_path, "wb") as f:
             f.write(asset["bytes"])
+    elif "directory" in asset:
```

Comment:
It seems more natural to check the value stored under the `"directory"` key than to check whether the `"directory"` key exists in the asset dictionary:
```suggestion
    elif asset.get("directory", False):
```